pkg/model/model_eks_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	 http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package model_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/aws-controllers-k8s/code-generator/pkg/testutil"
+)
+
+func TestEKS_IgnoreFieldPaths(t *testing.T) {
+	require := require.New(t)
+
+	g := testutil.NewModelForService(t, "eks")
+
+	crds, err := g.GetCRDs()
+	require.Nil(err)
+
+	crd := getCRDByName("Nodegroup", crds)
+	require.NotNil(crd)
+
+	// fields we have ignored via generator.yaml
+	ignoredFieldPaths := []string{
+		"Version",
+		"ScalingConfig.DesiredSize",
```

Comment:
Should'nt these be more `Nodegroup.Version` and `CreateNodegroupInput.ScalingConfig.DesiredSize` ?

Reply:
This is already under `getCRDByName("Nodegroup", ..)`, so logically it is now `Nodegroup.ScalingConfig.DesiredSize`.

But yeah, if we could rewrite it to a unit test then maybe it would look like that, or maybe we should test it through another code path than `getCRDByName`?